services/slam/builtin/cartographer_int_test.go

```diff
@@ -228,12 +260,18 @@ func integrationtestHelperCartographer(t *testing.T, mode slam.Mode) {
 		}
 	}
 
-	testCartographerPositionAndMap(t, svc)
+	testCartographerPosition(t, svc)
+	testCartographerMap(t, svc)
+
+	// Remove maps so that testing is done on the map generated by the internal map
+	test.That(t, resetFolder(name+"/map"), test.ShouldBeNil)
```

Comment:
I'm worried that we're losing test coverage for cartographer writing out maps itself. Can we test both running cartographer on the map in the map directory, and test running cartographer on the map returned by GetInternalState? Or IIRC, was there a flaky failure in running cartographer on the map in the map directory?

Reply:
I can add that test back in. I originally was fine changing it because we still tested loading the map created by the scheduled map_rate_sec during the localization test and I wanted to limit the length of the integration tests

Reply:
Ah, if we're still testing the map created by the scheduled map_rate_sec, then I think this is fine.